docs/pages/kresponsivewindow.vue

```diff
@@ -106,10 +187,24 @@
 
 <script>
 
+  import { computed } from '@vue/composition-api';
   import responsiveWindowMixin from '~~/lib/KResponsiveWindowMixin.js';
+  import useKResponsiveWindow from '~~/lib/useKResponsiveWindow.js';
 
   export default {
     mixins: [responsiveWindowMixin],
+    setup() {
+      const {
+        windowBreakpoint: _windowBreakpoint,
+        windowIsLarge: _windowIsLarge,
```

Comment:
If you wanted, I think you can destructure these fields without prefixing them with underscores. Just noting this because it isn't clear if there was a reason for this.

Reply:
The reason for prefixing these with underscores was to differentiate the mixin variables from the composable's(with _). For clarity, the composable variables will be renamed `composableWindowBreakPoint` and `composableWindowIsLarge` respectively.